packages/content-management/table_list_view_table/src/components/table.tsx

```diff
@@ -295,22 +272,33 @@ export function Table<T extends UserContentCommonSchema>({
       selectedUsers={tableFilter.createdBy}
       showNoUserOption={showNoUserOption}
     >
-      <EuiInMemoryTable<T>
-        itemId="id"
-        items={visibleItems}
-        columns={tableColumns}
-        pagination={pagination}
-        loading={isFetchingItems}
-        message={noItemsMessage}
-        selection={selection}
-        search={search}
-        executeQueryOptions={{ enabled: false }}
-        sorting={sorting}
-        onChange={onTableChange}
-        data-test-subj="itemsInMemTable"
-        rowHeader="attributes.title"
-        tableCaption={tableCaption}
-      />
+      <TagFilterContextProvider
+        isPopoverOpen={isPopoverOpen}
+        isInUse={isInUse}
```

Comment:
@sebelga, if I am not mistaken, we can simplify further because isInUse was a workaround for the problem that should be solved with context

Reply:
Needs to be checked but it could probably be removed indeed 👍